server/pages/index.jade

```diff
@@ -56,6 +56,10 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				link(rel='stylesheet', href=urls['style.css'])
 
 	body(class=isRTL ? 'rtl' : '')
+		noscript
+			p
```

Comment:
Maybe stick with plain text here and remove the p tag
<img width="553" alt="screen shot 2016-02-03 at 12 30 24 pm" src="https://cloud.githubusercontent.com/assets/1270189/12796110/0b655d22-ca72-11e5-8381-8ba45d3f0d93.png">